packages/kit/src/core/config/options.js

```diff
@@ -131,6 +135,9 @@ const options = object(
 					server: string(join('src', 'hooks.server')),
 					universal: string(join('src', 'hooks'))
 				}),
+				tracing: object({
+					server: string(join('src', 'tracing.server'))
+				}),
```

Comment:
We plan to deprecate the `kit.files` options in https://github.com/sveltejs/kit/pull/14152 so I'm not sure if we want to add `kit.files.tracing` if it will be removed soon.